pkg/plugins/backendplugin/plugin.go

```diff
@@ -0,0 +1,30 @@
+package backendplugin
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/infra/log"
+)
+
+// Plugin backend plugin interface.
+type Plugin interface {
+	PluginID() string
```

Comment:
Would be great if these functions could have comments.

Reply:
Will consider in future PR.